pandas/tests/groupby/test_index_as_string.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+from pandas.util.testing import assert_frame_equal, assert_series_equal
+import pandas.util.testing as tm
+
+
+def build_df_multi():
+    idx = pd.MultiIndex.from_tuples([('a', 1), ('a', 2), ('a', 3),
+                                     ('b', 1), ('b', 2), ('b', 3)])
+    idx.names = ['outer', 'inner']
+    df_multi = pd.DataFrame({"A": np.arange(6),
+                             'B': ['one', 'one', 'two',
+                                   'two', 'one', 'one']},
+                            index=idx)
+    return df_multi
+
+
+def build_df_single():
+    df_single = build_df_multi().reset_index('outer')
+    return df_single
+
+
+def build_test_series():
+    series_multi = build_df_multi().set_index('B', append=True)['A']
+    return series_multi
+
+
+class TestGroupByIndexAsString(object):
```

Comment:
no real need to make this a class, that is really a leftover from nose, just make these functions (of course a class is good for grouping generally)